evm/auxiliary/user_account_contract/contract.py

```diff
@@ -0,0 +1,157 @@
+from evm.constants import (
+    SECPK1_N,
+    ENTRY_POINT,
+)
+from evm.utils.hexadecimal import (
+    decode_hex,
+)
+from evm.utils.numeric import (
+    big_endian_to_int,
+)
+
+
+# S <= N // 2
+MAX_ALLOWED_S = SECPK1_N // 2
+VALIDATION_CODE_GAS = 3500
+GAS_RESERVE = 4500  # amount of gas reserved for returning
+GAS_RESERVE_OFFSET = 200
+NONCE_GETTER_ID = 0x141b5b48
```

Comment:
maybe add a comment to describe
`# NONCE_GETTER_ID = big_endian_to_int(keccak(b"get_nonce()")[:4])` ?